### PR TITLE
ordered by Födelsedatum by default

### PR DESCRIPTION
**Pull request by airenzp**

### CODE REVIEW

**Comment by jhagberg** [comment]
Its not sorting correctly if you have any rows with parningsdatum =!null

File: frontend/src/breeding_list.tsx [hunk 1]
```diff
@@ -123,6 +123,7 @@ export function BreedingList({ id }: { id: string | undefined }) {
             setActive(row);
           }}
           rowsPerPage={10}
+          orderBy="Födelsedatum"
```
Comment (HenrikeW) [inline frontend/src/breeding_list.tsx]: this needs to be `"birth_date"` instead of `"Födelsedatum"`.
Then it works.

**Review by HenrikeW — CHANGES_REQUESTED** [review]
Change one word, then it works :)

**Comment by HenrikeW** [comment]
It was such a small thing, so I added a commit for it

**Review by HenrikeW — APPROVED** [review]

**Comment by jhagberg** [comment]
WORKS!

File: frontend/src/breeding_list.tsx (hunk 1 above)
Comment (airenzp) [reply frontend/src/breeding_list.tsx]: ah, nice. I Dont know how it was working for me then.

**Pull request merged**